pkg/orchestrator/provider/common/utils.go

```diff
@@ -0,0 +1,109 @@
+// Copyright © 2023 Cisco Systems, Inc. and its affiliates.
+// All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package common
+
+import (
+	"fmt"
+	"math"
+
+	"github.com/sirupsen/logrus"
+
+	familiestypes "github.com/openclarity/vmclarity/pkg/shared/families/types"
+)
+
+// scanSizesGB represents the memory sizes on the machines that the tests were taken on.
+var scanSizesGB = []float64{0.01, 2.5, 8.1}
+
+// FamilyScanDurationsMap Calculate the logarithmic fit of each family base on static measurements of family scan duration in seconds per scanSizesGB value.
+// The tests were made on a t2.large instance with a gp2 volume.
+// The times correspond to the scan size values in scanSizesGB.
+// TODO add infoFinder family stats.
+// nolint:gomnd
+var FamilyScanDurationsMap = map[familiestypes.FamilyType]*LogarithmicFormula{
+	familiestypes.SBOM:             initFamilyScanDurationEstimators(scanSizesGB, []float64{0.01, 11, 37}),
+	familiestypes.Vulnerabilities:  initFamilyScanDurationEstimators(scanSizesGB, []float64{0.01, 1, 11}), // TODO check time with no sbom scan
+	familiestypes.Secrets:          initFamilyScanDurationEstimators(scanSizesGB, []float64{0.01, 720, 1320}),
+	familiestypes.Exploits:         initFamilyScanDurationEstimators(scanSizesGB, []float64{0, 0, 0}),
+	familiestypes.Rootkits:         initFamilyScanDurationEstimators(scanSizesGB, []float64{0, 0, 0}),
+	familiestypes.Misconfiguration: initFamilyScanDurationEstimators(scanSizesGB, []float64{0.01, 4, 5}),
+	familiestypes.Malware:          initFamilyScanDurationEstimators(scanSizesGB, []float64{0.01, 840, 2460}),
+}
+
+// LogarithmicFormula represents the formula y = a * ln(x) + b.
+type LogarithmicFormula struct {
+	a float64
+	b float64
+}
+
+// Extract returns receive an x value and returns the y value of the formula.
+func (lf *LogarithmicFormula) Extract(x float64) float64 {
+	return lf.a*math.Log(x) + lf.b
+}
+
+func initFamilyScanDurationEstimators(scanSizes, scanTimes []float64) *LogarithmicFormula {
```

Comment:
doesn't really matter what we evaluate here, I think it should be:
func initFamilyScanDurationEstimators(xData, yData []float64) *LogarithmicFormula {

Reply:
The idea was to create 2 functions, one to be used as a lib, and another specific to aws to be used during init. If you want to move it to aws then it's o.k. but it's a private function in the utils section that is used by AWS logic. it doesn't make sense